.github/workflows/deploy-prod.yaml

```diff
@@ -31,17 +31,17 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-nox-
 
-      - name: Set up Python 3.9
+      - name: Set up Python 3.8
         uses: actions/setup-python@v2
         with:
-          python-version: 3.9
+          python-version: 3.8
           architecture: x64
 
       - name: Install dependencies
         run: |
-          pip install poetry==1.1.4
-          pip install nox==2020.12.31
-          pip install nox-poetry==0.8.1
+          pip install poetry==1.1.11
```

Comment:
Kunne vi gått over til pipx i workflows:
```
pipx install poetry==1.1.11
pipx install nox==2021.10.1
pipx inject nox nox-poetry
```

Reply:
Kan sikkert det, men hva tjener vi på å gå over til pipx?

Reply:
Diskusjonen rundt dette fins her:
> A better fix would be to install Poetry and Nox in separate environments, using a tool like pipx.
ref https://github.com/cjolowicz/cookiecutter-hypermodern-python/issues/1050

I template-prosjektet sitt har cjolowicz gjort det på følgande måte:
```
      - name: Upgrade pip
        run: |
          pip install --constraint=.github/workflows/constraints.txt pip
          pip --version

      - name: Install Poetry
        run: |
          pipx install --pip-args=--constraint=.github/workflows/constraints.txt poetry
          poetry --version

      - name: Install Nox
        run: |
          pipx install --pip-args=--constraint=.github/workflows/constraints.txt nox
          pipx inject --pip-args=--constraint=.github/workflows/constraints.txt nox nox-poetry
          nox --version
```
der constraints.txt ser slik ut:
```
pip==21.3.1
nox==2021.10.1
nox-poetry==0.8.6
poetry==1.1.11
virtualenv==20.10.0
```
Eg har ikkje forstått alt av dette, men eg har ein tendens til å tenke at det han seier er lurt. 

Det er ellers greit for meg om du ikkje vil endre.

Reply:
constraints.txt er i hvert fall lurt å ta med, vi kan jo prøve pipx i tillegg